### ISSUE TITLE
Download options again

### ISSUE THREAD

**Issue by ericvd-ucb** [berkeley-dsep-infra/hubs#3696]
### Bug description

I am getting a request from EEP 118 about download options

Currently the recommendation is to do "pdf via html"
When I looked at it on Sept6th - there are multiple choices labeled pdf via html

The one on the bottom works
The one in the middle doesn't work

Maybe the one on the bottom used to be called "pdf via Chrome" ?

### Environment & setup

- Hub: 
R Hub / Jupyter R kernel
- Language: 
Pandoc/Latex maybe

### How to reproduce

I pulled from the link at 
https://github.com/ds-modules/ENVECON-118-FA22

Navigated to the notebook for SA1

then checked Download options
the one in fifth position didnt work
the one in last position did work

**Comment by ericvd-ucb** [comment]
And a further mess - the download options are different on RETRO !!

Looks to me like RETRO has more possible options 
and guessing by the font - looks to me like the pdf option is PDF via Latex
not PDF via HTML or formerly PDF via Chrome
which is our recommended option

I am wondering if we could
1. reduce the number of options
2. resptore something like "pdf via Chrome" to the current situation of duplicate "pdf via html" options
3. add "pdf via chrome" to RETRO

**Comment by balajialg** [comment]
@ericvd-ucb Couple of things,

1. Jupyter notebook 7.0 release will make retro the default interface instead of the classic notebook, Will this solve for your recommendation 1 and 2?
2. Also, "File -> Save and Export Notebook as -> PDF or web pdf" options generate PDF files. Do you still think a "PDF via chrome" option needs to get added to the retro UI?

**Comment by ericvd-ucb** [comment]
On 1 @balajialg - default interface for UC Berkeley ?  Im OK if that is the policy decision we are making but Lets ramp up communication if that is the case. I think we have plenty of users on classic who may not know about this .  And default is also up to nbgitpuller link - anyone can make any link, or use any other link. If we need to migrate people on purpose or redirect to NB7 only ... then we need a plan for that 

As an aside it has come to my attention recently that retro is a bad branding name for NB7.   Retro is the new interface and Classic is the old interface is a problematic 

2) Ok looking at this one more time within NB7
I would be interested to have a more streamlined set of download options - changing UI
but not necessary if needed  we can keep it as is
To me it looks like 
the one called PDF is the old "PDF via Latex"  ~ which historically coudl be buggy if Latex is problemeatic
the one called WebPDF is the old "Pdf via Chrome" ~ which was more consistent 

I guess my question could be rephrased as what is our - UC Berkeley instructor -  recommended path for consistent behavior in downloads

**Comment by ryanlovett** [comment]
> As an aside it has come to my attention recently that retro is a bad branding name for NB7. Retro is the new interface and Classic is the old interface is a problematic

Fwiw, though retro will become notebook 7, the "retro" name will go away. People won't see "retro", if that is a concern.

**Comment by balajialg** [comment]
@ericvd-ucb Yes, we need to plan our communication for this NB 7.0 migration. The open question is when will the stable version of NB 7.0 gets released. Assuming this release will happen before Spring 23, I will draft the corresponding message and update our documentation to reflect these changes before the Spring semester. 

It seems like a good next step to go with Save and Export Notebook as WebPDF for those using retrolab currently given our historical issues with latex? There is also a [7-month-old issue](https://github.com/jupyter/notebook/issues/6296) filed in the Jupyter notebook repository which highlights the 500 internal server error issue caused due to downloading via the Latex option.

**Comment by balajialg** [comment]
Relevant issues reporting challenges with downloading notebooks - https://github.com/berkeley-dsep-infra/datahub/issues/4112 and https://github.com/jupyter/nbconvert/issues/1938
